tests/includes/colors.sh

```diff
@@ -1,11 +1,7 @@
 supports_colors() {
-	if [[ -z ${TERM} ]] || [[ ${TERM} == "" ]] || [[ ${TERM} == "dumb" ]]; then
-		echo "NO"
-		return
-	fi
```

Comment:
I do remember I ran into problems with tput and determining if colors were supported. I wonder if this would have been better with a separate PR though.

Reply:
Apparently it's not the main issue ^^ (shfmt continue to fails, but this have fixed the color issue).

Will push another PR to fix that stuff, to allows Jordan to helps (seems to be his changes)